### ISSUE TITLE
Link to pkg.go.dev from our website

### ISSUE THREAD

**Issue by klesh** [merico-dev/lake-website#54]
Beside official documentation / guides , we have some `godoc` from `source files` which are more in-depth documentation to help develop understand and utilize our built-in `function`/ `helper`.
`godoc` is great because it would pop-up automatically  in IDE during development. And they would be generated automatically by pkg.go.dev without any effort, so other can browse our `godoc` by browser and search doc they needed.

In short, we have to add a link to https://pkg.go.dev/github.com/apache/incubator-devlake from official website

**Comment by Palanikannan1437** [comment]
Hey @klesh, I'd like to do this...it's a wonderful resource indeed!

Could you please assign it to me and guide me as to where/which section we'd need to add this link in our website?

**Comment by klesh** [comment]
Thanks @Palanikannan1437 .
I think the [Developer Setup](https://devlake.apache.org/docs/next/DeveloperManuals/DeveloperSetup) would be the appropriate place to be.

**Comment by klesh** [comment]
We could add a section like References

**Comment by Palanikannan1437** [comment]
> We could add a section like References

Ohh okay, got it @klesh 😄
